<launch>
  <node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" />

  <node name="ardrone_tracker" pkg="ardrone_autonomy" type="ardrone_tracker.py" >
    <param name="cascadefile" value="$(find ardrone_autonomy)/cascade/haarcascade_falkorlogotshirt.xml" />
  </node>

  <node name="extract_images" pkg="image_view" type="extract_images">
    <remap from="image" to="/ardrone/front/image_raw" />
  </node>

  <node name="ardrone_teleop_joy" pkg="ardrone_autonomy" type="ardrone_teleop_joy.py" />

  <param name="joy_node/dev" value="/dev/input/js1" />
  <node name="joy_node" pkg="joy" type="joy_node" />
<!--  <node name="record" pkg="rosbag" type="record" args="-e /ardrone/front/image_raw" /> -->

<!--  <node pkg="rxtools" type="rxplot" name="plot_cmd" args="/cmd_vel/linear/x:y:z,/cmd_vel/angular/z" /> -->
<!--  <node pkg="rxtools" type="rxplot" name="plot_pos" args="/ardrone_tracker/found_point/x:y:z" /> -->
  <node pkg="rxtools" type="rxplot" name="plot_battery" args="/ardrone/navdata/batteryPercent" />

</launch>
